### PR TITLE
[daint dom] Adding GREASY to production

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage = 'https://github.com/jonarbo/GREASY'
+description = """Greasy is a tool designed to make easier the deployment of
+embarrassingly parallel simulations in any environment. It is able to run in parallel a list of different tasks."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = ['/apps/common/easybuild/sources/g/GREASY/greasy-%(version)s.tar.gz']
+
+dependencies = [
```
Comment (jgphpc) [inline easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: single line ?

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'GREASY'
+version = '2.1-cscs'
+
+homepage = 'https://github.com/jonarbo/GREASY'
+description = """Greasy is a tool designed to make easier the deployment of
```
Comment (jgphpc) [inline easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: too wide

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+embarrassingly parallel simulations in any environment. It is able to run in parallel a list of different tasks."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = ['/apps/common/easybuild/sources/g/GREASY/greasy-%(version)s.tar.gz']
```
Comment (jgphpc) [inline easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: can you github it ?

Comment (victorusu) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: nope. Not this version...

Comment (victorusu) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: we haven't contributed everything, yet

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb (hunk 2 above)
Comment (jgphpc) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: thks

File: easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb (hunk 3 above)
Comment (jgphpc) [reply easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-17.08.eb]: ok next time

**Pull request merged**